### DISCUSSION TITLE
Attribution reports come empty

### DISCUSSION THREAD

**Discussion by 0Viralix** [amzn/ads-advanced-tools-docs#46] [General]
Hi,
We are building new tool based on the Attribution Reports.
Unfortunately, we can't get the information.

We have all the info, we get 200OK response,
But the report comes empty.

Anyone else meet this issue?

**Comment by elizafarley** [comment]
Hi @0Viralix Are you able to see attribution data for advertisers in the [advertising console attribution UI](https://advertising.amazon.com/attribution)? Looking to understand whether this could be an attribution set up issue or something else.
